config.toml

```diff
@@ -8,6 +8,7 @@ disableKinds = []
 themesDir = "node_modules/"
 enableRobotsTXT = true
 pluralizeListTitles = false
+buildFuture = true # A new event website will always be in the future
```

Comment:
This should not be there.

Reply:
See: https://stackoverflow.com/questions/59655470/hugo-date-vs-publishdate#:~:text=Date%20%2D%20the%20date%20associated%20with,the%20content%20was%20last%20modified

Reply:
removed